content/version/1/4/code-of-conduct.id.md

```diff
@@ -0,0 +1,68 @@
++++
+version = "1.4"
+aliases = ["/version/1/4"]
++++
+
+# Kode Etik Contributor Covenant
+
+## Janji Kami
+
+Dalam minat kami untuk membuat lingkungan yang terbuka dan ramah, kami sebagai 
```

Comment:
please consider 'sesuai minat kami' to make it smoother to read rather than 'dalam minat kami'.